<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dev_UI</class>
 <widget class="QWidget" name="Dev_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1535</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Set_filter_coeffs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Set Filter Coefficients in s-domain (4-order Butterworth Filter)</string>
   </property>
   <widget class="QLabel" name="lbl_numerator">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numerator:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_denominator">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Denominator:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_nume_order_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_nume_order_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_nume_order_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_nume_order_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_order_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>order 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_order_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>order 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_order_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>order 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_order_1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>order 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coefficients">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Coefficients:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_nume_order_0">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_order_0">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>order 0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_deno_order_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_deno_order_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_deno_order_0">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_deno_order_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_deno_order_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_filter_coeffs">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set coefficients</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Filtered_PPG">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>821</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtered PPG signal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_switch_to_user_ui">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>740</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch to User UI</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Raw_PPG">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>821</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Raw PPG signal</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Set_RTC">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Set RTC</string>
   </property>
   <widget class="QComboBox" name="cbb_set_day">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Monday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thursday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Friday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saturday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sunday</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_set_rtc">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set time</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_mode_time">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Date time</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Epoch time</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_epoch_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Epoch time (GMT0):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mode_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_epoch_time">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date time (GMT7):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_day">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Day:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_set_date">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_set_month">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_month">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Month:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_set_year">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_year">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_set_hour">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hour">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hour:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_set_min">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_min">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sec">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sec:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_set_sec">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Notification">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Notification</string>
   </property>
   <widget class="QLabel" name="lbl_notification">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_notification">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>491</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
